cli/cmd/clean.go

```diff
@@ -0,0 +1,154 @@
+package cmd
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/apex/log"
+	"github.com/spf13/cobra"
+	"github.com/tarantool/tt/cli/cmdcontext"
+	"github.com/tarantool/tt/cli/configure"
+	"github.com/tarantool/tt/cli/modules"
+	"github.com/tarantool/tt/cli/running"
+)
+
+var forceRemove bool
+
+// NewCleanCmd creates clean command.
+func NewCleanCmd() *cobra.Command {
+	var cleanCmd = &cobra.Command{
+		Use:   "clean [INSTANCE_NAME]",
+		Short: "Clean instance(s) files",
+		Run: func(cmd *cobra.Command, args []string) {
+			err := modules.RunCmd(&cmdCtx, cmd.Name(), &modulesInfo, internalCleanModule, args)
+			if err != nil {
+				log.Fatalf(err.Error())
+			}
+		},
+	}
+
+	cleanCmd.Flags().BoolVarP(&forceRemove, "force", "f", false, "do not ask for confirmation")
+
+	return cleanCmd
+}
+
+func collectFiles(list *[]string, dirname string) error {
```

Comment:
Why not. API is simpler.

Reply:
I'm not very good at Go and can't tell if this is an idiomatic approach or not (and I don't think I'm the only one). I think it would be nice to add a link to the source of idiomatic (link to examples in popular repositories, code from a standard library, ...).

Reply:
fixed